arcgis-ios-sdk-samples/Content Display Logic/Controllers/Color Picker/ColorPickerViewController.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+class ColorPickerViewController: UITableViewController {
+    
+    @IBOutlet var hueSlider: UISlider!
+    @IBOutlet var saturationSlider: UISlider!
+    @IBOutlet var brightnessSlider: UISlider!
+    
+    @IBOutlet var hueLabel: UILabel!
+    @IBOutlet var saturationLabel: UILabel!
+    @IBOutlet var brightnessLabel: UILabel!
+    
+    private var color: UIColor {
+        set {
+            newValue.getHue(&hue, saturation: &saturation, brightness: &brightness, alpha: &alpha)
+        }
+        get {
+            return UIColor(hue: hue, saturation: saturation, brightness: brightness, alpha: alpha)
+        }
+    }
+    private var onUpdate: ((UIColor)->Void)? = nil
```

Comment:
I would encourage you to reconsider using a delegate callback for this. It takes a little more work to create the protocol, but it is a very common pattern. Using a closure property like this still feels odd to me.